src/hammer-vlsi/par/mockpar/defaults.yml

```diff
@@ -0,0 +1,4 @@
+par.mockpar:
+  # A folder where mocksynth can create files.
+  temp_folder: null
```

Comment:
Is this necessary? Can't we just use python's normal temp dir creation mechanism?

Reply:
I'll fix the comment but just leave this in to be the same as mocksynth.